src/coreclr/src/tools/ReadyToRun.SuperIlc/BuildFolder.cs

```diff
@@ -1,4 +1,4 @@
-// Licensed to the .NET Foundation under one or more agreements.
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
We prefer files without BOM.

Reply:
OK, will remove. Is there a way to make VS not automatically add them everywhere?

Reply:
Fixed in 2nd commit.